codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowControllerTest.java

```diff
@@ -290,7 +311,7 @@ private int window(int streamId) throws Http2Exception {
         return controller.windowSize(stream(streamId));
     }
 
-    private Http2Stream stream(int streamId) throws Http2Exception {
```

Comment:
This was an artifact of the work I was doing regarding stream history. Reverted.